app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @like = @post.likes.build(user_id: params[:user_id])
```

Comment:
- [ ] Instead of using the `user_id: params[:user_id]`, please consider using the `current_user` method you created earlier, this is because if for example the `current_user` likes a post from a user with id `5`, the like will be saved on behalf of the user `5` instead of the expected `current_user`. 🤓 🔧

Reply:
Thank you